rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -1,45 +0,0 @@
----
-# Copyright 2014-2017, Rackspace US, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-container_skel:
-  nova_api_metadata_container:
-    properties:
-      service_name: nova
-      lxc_container_variant: "nova-{{ rpc_release }}"
```

Comment:
The container variant bits are still required until https://github.com/rcbops/rpc-openstack/pull/2375 is finalised and merged, so this content must stay.

The only thing that should be removed is the group membership changes below.